configs/server/forseti_conf_server.yaml.in

```diff
@@ -421,6 +421,9 @@ notifier:
         organization_id: {ROOT_RESOURCE_ID}
         # gcs_path should begin with "gs://"
         gcs_path:
+        # Cloud SCC Beta API uses a new source_id.  It is unique per
```

Comment:
Might be confusing to user on whether or not source_id is part of mode api and which mode is being used if organization_id and source_id are both filled out.

Reply:
Added additional comment to say that beta api will take precedence.  Or else, please give me a concrete suggestion on how to improve this.